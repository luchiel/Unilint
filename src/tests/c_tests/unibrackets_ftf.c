int a[] = [5, 4, 3];
int b[] = [ 5, 4, 3 ];

int main() { return 0; }
int not_main() {return ( 0 );}
