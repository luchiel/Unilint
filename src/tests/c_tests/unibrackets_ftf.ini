[spaces_and_newlines]
spaces_inside_braces = false
spaces_inside_curly_braces = true
spaces_inside_brackets = false
