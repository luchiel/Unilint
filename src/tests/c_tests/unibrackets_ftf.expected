2:12: unexpected space after [
2:20: unexpected space before ]
5:17: no space after {
5:25: unexpected space after (
5:27: unexpected space before )
5:29: no space before }
